schematics/collection.json

```
@@ -0,0 +1,9 @@
+{
+    "$schema": "../node_modules/@angular-devkit/schematics/collection-schema.json",
+    "schematics": {
+      "nest-add": {
+        "description": "Schematics to add In memory DB module.",
+        "factory": "./nest-add/"
```

You missed the schema here
```suggestion
        "factory": "./nest-add/",
        "schema": "./nest-add/schema.json"
```